.travis.yml

```diff
@@ -26,5 +26,4 @@ script:
   - goveralls -service=travis-ci -v -package ./pkg/... -ignore "pkg/client/*/*.go,pkg/client/*/*/*.go,pkg/client/*/*/*/*.go,pkg/client/*/*/*/*/*.go,pkg/client/*/*/*/*/*/*.go,pkg/client/*/*/*/*/*/*/*.go,pkg/apis/mxnet/*/zz_generated.*.go"
 
 notifications:
```

Comment:
We can remove the section directly if we do not have any webhooks.

Reply:
Good idea. Done